docs/releases/4.2.txt

```diff
@@ -48,6 +48,9 @@ Minor features
   :attr:`~django.contrib.admin.ModelAdmin.filter_vertical` widgets are now
   filterable.
 
+* The ``autofocus`` attribute was removed from the search box due to accessibility
+   considerations.
```

Comment:
ticket-34040 was classified as a bug, so release notes are unnecessary.

Reply:
Also my bad - it seemed useful to me in case someone wondered why it wasn't working anymore - it is a noticeable change in behaviour.

Reply:
We can leave it if you think it's helpful, but it should be moved to the _"Backwards incompatible changes in 4.2 -> Miscellaneous"_.

Reply:
Sounds reasonable, will do